requirements/base.in

```diff
@@ -4,7 +4,7 @@ celery-once
 defusedxml
 furl
 glom
-git+https://github.com/maykinmedia/json-logic-py.git@1ceb93ad165d69a5639edc623d5613d0229c05e2#egg=maykin-json-logic-py
+git+https://github.com/maykinmedia/json-logic-py.git@28501d7d4ed9012554d0b82ef340f1db6d869434#egg=maykin-json-logic-py
```

Comment:
the PR has been merged, so this commit hash should probably be updated to the merge commit on the main branch